### ISSUE TITLE
Kabanero built-in template source should be marked as 'protected' 

### ISSUE THREAD

**Issue by tetchel** [eclipse/codewind#480]
The user should not be allowed to delete the Kabanero template source, same as Codewind and Appsody.

[code](https://github.com/eclipse/codewind/blob/master/src/pfe/portal/modules/Templates.js)
```
const KABANERO_REPO = {
  url: 'https://github.com/kabanero-io/collections/releases/download/v0.1.2/kabanero-index.json',
  description: 'Kabanero Collections',
  enabled: false,
};
```

**Comment by tetchel** [comment]
this can be fixed along with #444

**Comment by tetchel** [comment]
I removed the hot tag but I still think this bears discussion for 0.4 because it would be so annoying to a user
